transform breaks when building on windows

<!-- Before creating an issue please make sure you are using the latest version of style-loader. -->

**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
When building with Webpack on windows, each css fails with
`Module not found: Error: Cannot resolve module 'C:folderNamesrccssVariablesPolyfill'`
(the path to the transform module, stripped from slashes)
The tests also fail on Windows.

**If the current behavior is a bug, please provide the steps to reproduce.**
Run tests on Windows, or a Webpack config with a tranform module for style-loader.

**What is the expected behavior?**
It should not fail when building on Windows.

**Please mention other relevant information such as your webpack version, Node.js version and Operating System.**
Webpack 1.14.1
Node 5.0.0
Windows 10

--- a/index.js
+++ b/index.js
@@ -16,7 +16,7 @@ module.exports.pitch = function(remainingRequest) {
 		"if(typeof content === 'string') content = [[module.id, content, '']];",
 		"// Prepare cssTransformation",
 		"var transform;",
-		query.transform ? "transform = require('" + path.resolve(query.transform) + "');" : "",
+		query.transform ? "transform = require(" + loaderUtils.stringifyRequest(this, "!" + path.resolve(query.transform)) + ");" : "",
 		"var options = " + JSON.stringify(query),
 		"options.transform = transform",
 		"// add the styles to the DOM",

